web/modeltr/base.py

```diff
@@ -57,3 +58,8 @@ class trMachineState:
 class trRequestState:
     _default = RequestState.CREATED
     _type = RequestState
+
+
+class trRequestType:
+    _default = '<null>'
```

Comment:
some better hint for default? Honestly, I do not want any default for this type...but is there a way how to achieve that?

Reply:
I suppose it is used somewhere in document.py so it cannot be simly omitted

Reply:
So do I understand correctly that I can delete whole like with `_default = '<null>'` ?